robot/impl/local_robot.go

```diff
@@ -192,6 +192,12 @@ func (r *localRobot) Close(ctx context.Context) error {
 	return err
 }
 
+// Kill will attempt to kill any processes on the system started by the robot as quickly as possible.
+// This operation is not clean and will not wait for completion.
+func (r *localRobot) Kill() {
+	r.manager.Kill()
```

Comment:
It feels a little awkward that localRobot.Kill only calls kill on the resource manager.

And the resource manager only calls kill on the mod manager.

And localRobot already has a handle on the modmanager. So why doesn't it call kill directly? Or just have the part that's about to do the log.Fatal/os.Exit call kill on the modmanager?

Reply:
keeping the abstractions I think is better in the long run - the resource manager will eventually call kill on the process manager. 
the part that's about to do log.Fatal doesn't have a handle to the modmanager, since it only has access to the LocalRobot interface.

Reply:
> keeping the abstractions I think is better in the long run

If I'm adding some plumbing from local robot -> modmanager, when should I use `localRobot.modmanager` directly and when should I add additional plumbing through `localRobot.manager`?

Reply:
FWIW, I'm fine with leaving this as-is. The above is me just coming to the realization that there's some abstraction that I didn't know existed.

Reply:
> localRobot.modmanager

I'm not aware of an existing `localRobot.modmanager` field? Or are you suggesting adding one?

Reply:
I think @dgottlieb is asking whether he should use `localRobot.manager.modmanager`.

To @dgottlieb's question, I don't have a good answer. I feel like localRobot and resource manager are already a bit mangled, and I think adding the plumbing will end up making the unmangling slightly more manageable if we ever decide to do it